helm/external-dns-app/values.yaml

```
@@ -2,203 +2,20 @@
-  aws_secret_access_key:
-
-  # externalDNS.domainFilterList
-  # One or more domains that this instance of external-dns will manage. If
-  # no domains are provided then the cluster baseDomain must be provided via
-  # either `baseDomain` or `aws.baseDomain` (depending on the Route53 access
-  # method used).
-  # By explicitly setting externalDNS.domainFilterList to null, the --domain-filter
-  # flag will be completely omitted. Otherwise leave this as empty array ([])
-  domainFilterList: []
-  # - foo.bar.com
-  # - baz.bar.com
-
-  # externalDNS.dryRun
-  # Dry run logs what action external-dns would have taken, but doesn't apply them.
-  dryRun: false
-
-  # externalDNS.interval
-  # Interval between synchronisations. Must be in duration format (e.g. `5m`)
-  interval: 5m
-
-  # externalDNS.namespaceFilter
-  # The namespace to limit sources of endpoints to. If left blank, all
-  # namespaces will be used.
-  namespaceFilter: kube-system
-
-  # externalDNS.minEventSyncInterval
-  # The minimum interval between two consecutive synchronizations triggered from
-  # kubernetes events in duration format (default: 5s)
-  minEventSyncInterval: 30s
-
-  # externalDNS.policy
-  # Syncing policy between sources and providers.
-  policy: sync
-
-  # externalDNS.registry
-  # Configuration options relating to ownership of created records.
-  registry:
-
-    # externalDNS.registry.txtOwnerID
-    # String to identify this instance of external-dns and the records it owns. See
-    # https://github.com/kubernetes-sigs/external-dns/blob/master/docs/proposal/registry.md
-    txtOwnerID: managed-external-dns
-
-    # externalDNS.registry.txtPrefix
-    # String prefix applied to TXT registry records. See
-    # https://github.com/kubernetes-sigs/external-dns/blob/master/docs/proposal/registry.md
-    txtPrefix: extdns
-
-  # externalDNS.sources
-  # Resource types queried for endpoints.
-  sources:
-  - service
-  # Disable ingress source by default.
-  # - ingress
-
-  # externalDNS.extraArgs
-  # Allows to specify additional arguments to the external-dns deployment.
-  extraArgs: []
-
-# set the HTTP_PROXY, HTTPS_PROXY and NO_PROXY variable
-proxy:
-  noProxy:
-  http:
-  https:
-cluster:
-  # is getting overwritten by the top level proxy if set
-  # These values are generated via cluster-apps-operator
-  proxy:
-    noProxy:
-    http:
-    https:
-
-# crd
-# Configuration options for CRDs.
-crd:
-
-  # crd.backoffLimit
-  # setting this higher means the CRD creation is less likely to be marked
-  # as failed.
-  backoffLimit: 10
-
-  # crd.image
-  image:
-
-    # crd.image.pullPolicy
-    pullPolicy: IfNotPresent
-
-  # crd.install
-  # CRDs are installed by default. If this is disabled then CRDs must
-  # be managed by another method.
-  install: true
+annotationFilter: "giantswarm.io/external-dns=managed"
 
-  # crd.resources
-  resources:
+image:
+  name: giantswarm/external-dns
+  registry: docker.io
```

```suggestion
  registry: gsoci.azurecr.io
```
See https://github.com/giantswarm/roadmap/issues/3040 
Doing this in a separate PR is also fine

---

Yeah, coming in another PR. I won't release v3 just yet :)